src/libmusig2.h

```diff
@@ -9,16 +9,20 @@
 #include "random.h"
 
 
-// todo: use the secp library constants
-#define V                               2           // Number of nonce values
-#define SCALAR_BYTES                    32          // SCALAR BYTES
-#define XONLY_BYTES                     32          // XONLY PUBLIC KEY BYTES
-#define PAR_SIG_BYTES                   32          // PARTIAL SIGNATURE BYTES
-#define PK_BYTES                        64          // FULL SIZE PUBLIC KEY BYTES
-#define SCH_SIG_BYTES                   64          // SCHNORR SIGNATURE BYTES
-#define SER_PK_BYTES_COMPRESSED         33          // Serialized public key bytes for compressed version
+#define V                                   2           // Number of nonce values
+#define MUSIG2_SCALAR_BYTES                 32          // SCALAR BYTES
+#define MUSIG2_PARSIG_BYTES                 32          // PARTIAL SIGNATURE BYTES
+#define MUSIG2_AGGR_PUBKEY_BYTES            32          // XONLY PUBLIC KEY BYTES
+#define MUSIG2_PUBKEY_BYTES_COMPRESSED      33          // COMPRESSED PUBLIC KEY BYTES
+#define MUSIG2_PUBKEY_BYTES_FULL            64          // FULL SIZE PUBLIC KEY BYTES
+#define MUSIG2_BYTES                        64          // SCHNORR SIGNATURE BYTES
 
```

Comment:
I'm not convinced by this naming. But this is just a suggestion. You'll be the main maintainer of the lib, so you can make the final call. The following is my reasoning: 
* `MUSIG2_PARSIG_BYTES` is a bit confusing, because we define `musig2_partial_signature` to be two values with a total of 64 bytes. This variable is really used to determine the size of the 'scalar part' of the partial signature.
* I think I would remove the parsig bytes, and simply use`SCALAR_BYTES` and `MUSIG2_BYTES` for referring to the size of the signatures.